src/blocks/Media/Media.tsx

```diff
@@ -16,7 +16,7 @@ export const MediaBlock = (props: MediaBlockProps) => {
     return (
         <MediaBase {...props} onScroll={() => setPlay(true)}>
             <MediaBase.Card>
-                <Media {...mediaThemed} playVideo={play} />
+                <Media {...mediaThemed} playVideo={play} border={border} rounded />
```

Comment:
What is `rounded`?

Reply:
PR was completely refactored. In discussion we came up with the solution that there should be an mixin added to media component